Tests/PortToTripleSlash/TestData/Basic/SourceExpected.cs

```diff
@@ -168,5 +168,32 @@ public void MyTypeParamMethod<T>(int param1)
         {
             return value1;
         }
+
+        /// <summary>This is the <see cref="MyNamespace.MyType.Enumerator" /> class summary.</summary>
+        internal class Enumerator
```

Comment:
Only `public` and `protected` APIs are backported. I'm not sure why the tests are still passing, they shouldn't.

Same applies for the other two `internal` APIs added below. Can you set them to `public`? I'll investigate why the tests did not complain.

Reply:
I'll set them to public. I'm still investigating too. Unexpectedly, the tests are passing without my fix at all, so there's something about the types in runtime that this fixes that isn't captured in the tests. I'll continue working on it, but we shouldn't merge this until I have tests that fail before the fix.